src/main/java/Duke.java

```diff
@@ -1,10 +1,48 @@
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        TaskManager taskManager = new TaskManager();
+
+        initialDisplay();
+
```

Comment:
I like how you make the main function neater by making smaller callable function